### PR TITLE
Univers

### PR DESCRIPTION
**Pull request by jeremyjorand**

### CODE REVIEW

**Review by Hamza-Luciano — DISMISSED** [review]
add univers

**Review by LuigiMenez — APPROVED** [review]
resolve a conflict

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Resolve a conflict

File: frontend/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=100Vw, initial-scale=1.0" />
```
Comment (LoicBrassart) [inline frontend/index.html]: Why?

**Review by Hamza-Luciano — DISMISSED** [review]
modification

File: frontend/src/components/Univers/style.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import styled from "styled-components";
+
+export default styled.div`
+  color: white;
+  .main_container {
```
Comment (LoicBrassart) [inline frontend/src/components/Univers/style.jsx]: snake_case, camelCase, kebab-case : I saw these three cases in your classes' names.
Chose one and fix it everywhere please

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Fix some Evils

File: frontend/index.html (hunk 1 above)
Comment (ThomasDCW) [reply frontend/index.html]: because we can't use style reset

**Review by Hamza-Luciano — APPROVED** [review]
ok for us

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**